### PR TITLE
Feature issue template

### PR DESCRIPTION
**Pull request by billgeo**
Fixes: # 114

### Change Description:
- Added a github template for new features.
- Improved formatting of other templates

#### Source Code Documentation Tasks:
- n/a README updated (where applicable)
- n/a CHANGELOG updated

#### User Documentation Tasks:
- n/a Confluence user guide updated (where applicable)

#### Testing Tasks:
- n/a Added tests that fail without this change
- n/a All tests are passing in development environment
- [x] Reviewers assigned

#### Pull Request Management:
- [ ] Linked to sub-task
- n/a Linked to the epic

### CODE REVIEW

**Comment by billgeo** [comment]
Doing this because talked to NZGB today and they have some new features they want. Suggested they put all this down in a ticket and template will help.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -4,21 +4,34 @@ about: Create a report to help us improve
...
 ---
 
+<!--
+Checklist before submitting
+
+- [ ] Search through existing issue reports to check whether the issue already exists
+- [ ] Ensure you are running the latest software release, see https://github.com/linz/gazetteer/releases
+- [ ] Check QGIS/plugin log messages for more information.
+- [ ] If you can please create a light and self-contained sample dataset to reproduce the issue.
+-->
```
Comment (SPlanzer) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: It appears commented out sections do not render. Perhaps just provide the checklist?

Comment (SPlanzer) [reply .github/ISSUE_TEMPLATE/bug_report.md:14]: ignore that. I see what you are doing here. It is a good idea.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -4,21 +4,34 @@ about: Create a report to help us improve
...
+
 ### Bug Description
-A clear and concise description of what the bug is and what you expected to happen.
+<!-- A clear and concise description of what the bug is and what you expected to happen. -->
 
 #### Steps to Reproduce
-Steps to reproduce the behavior:
+<!-- Steps to reproduce the behavior -->
+
 1. Go to '...'
```
Comment (SPlanzer) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: should this example also be commented out?

Comment (billgeo) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: Was thinking that bit was more like something the user would edit with their own information. So could leave it uncommented. I guess the principle is to minimise the amount of typing/deleting they need to do.

Comment (SPlanzer) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: sounds sound.

**Pull request merged**